src/Microsoft.AspNet.Authentication.OpenIdConnect/OpenIdConnectMiddleware.cs

```diff
@@ -91,10 +91,7 @@ public class OpenIdConnectMiddleware : AuthenticationMiddleware<OpenIdConnectOpt
                 throw new ArgumentException("Options.SignInScheme is required.");
             }
 
-            if (Options.HtmlEncoder == null)
-            {
-                Options.HtmlEncoder = services.GetHtmlEncoder();
-            }
+            HtmlEncoder = services.GetHtmlEncoder();
```

Comment:
good point, you don't need to null check stuff that's directly passed to base, that with execute first anyways.